types/file-loader/index.d.ts

```diff
@@ -1,9 +1,8 @@
-// Type definitions for file-loader 4.2
+// Type definitions for file-loader 6.2
```

Comment:
```suggestion
// Type definitions for file-loader 5.0
```

`esModule` was added in v5, so it would be good to match that for this change :)